attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AuditableAttributeMaskingResponse.java

```diff
@@ -42,8 +42,7 @@ private AuditableAttributeMaskingResponse(
     }
 
     /**
-     * Chain any errors from previous stream elements
-     *
+     * Chain any errors from previous stream elements.
```

Comment:
Could the extra line be added back in so there is a gap between description and the params?
Also check this on lines 80 and 88

Reply:
done